src/components/VariableEditModal.vue

```diff
@@ -60,7 +57,15 @@
 
   const api = useWorkspaceApi()
 
-  const isUnique: ValidationRule<string | undefined> = async (value, label, { signal, source, previousValue }) => {
+  const valueIsLessThanOrEqual: ValidationRule<string | undefined> = (value) => {
+    if (isNull(value) || !isString(value)) {
+      return false
+    }
```

Comment:
So since you've defined your rule as `ValidationRule<string | undefined>` I'd change this to 
```
if(!isDefined(value)) {
  return false
}
```